### PR TITLE
Improve formatting of gauges.

### PR DESCRIPTION
**Pull request by mixalturek**
Rebase + new pull request to simplify review.

### CODE REVIEW

File: formatting/src/test/java/com/avast/metrics/dropwizard/formatting/FormattingMetricsMonitorTest.java [hunk 1]
```diff
@@ -107,4 +110,35 @@ public void testFormatTwoCounters() throws Exception {
...
+                    "gauge.boolean 1\n" +
+                    "gauge.byte 2\n" +
+                    "gauge.double 7.1\n" +
+                    "gauge.float 7.0\n" +
+                    "gauge.int 4\n" +
+                    "gauge.long 5\n" +
+                    "gauge.null null\n" +
+                    "gauge.short 3";
+
+            assertEquals(expected, monitor.format());
```
Comment (jendakol) [inline formatting/src/test/java/com/avast/metrics/dropwizard/formatting/FormattingMetricsMonitorTest.java]: When I look at this call, wouldn't it be better to name the method `formatted` or `getFormatted`? Just idea... :thinking:

Comment (mixalturek) [inline formatting/src/test/java/com/avast/metrics/dropwizard/formatting/FormattingMetricsMonitorTest.java]: I don't have any preference here. I little more prefer simple `format()` - there are neither `mapped()`, `filtered()`, `flatMapped()`, nor `getMapped()`, etc.

Comment (jendakol) [inline formatting/src/test/java/com/avast/metrics/dropwizard/formatting/FormattingMetricsMonitorTest.java]: I know, however these method you've named always take some parameter (as usually does `format` method too, like on `String` in Scala).  
@jakubjanecek what do you think?

Comment (jakubjanecek) [inline formatting/src/test/java/com/avast/metrics/dropwizard/formatting/FormattingMetricsMonitorTest.java]: I guess I don't really care about this one :) `format` seems ok to me.

**Pull request merged**